Orange/evaluation/testing.py

```diff
@@ -365,8 +365,10 @@ def _is_picklable(obj):
         recursion ensues.
 
         Guard your executed code with above Python idiom, or pass n_jobs=1
-        to evaluation methods, i.e. {}(..., n_jobs=1).
-            '''.format(self.__class__.__name__)) from None
+        to evaluation methods, i.e. {}(..., n_jobs=1). Setting n_jobs to 1.
+            '''.format(self.__class__.__name__), OrangeWarning)
+        mp_queue = mp.Queue()
+        n_jobs = 1
```

Comment:
These two lines are probably under-indented. They should be under `except`. Perhaps move them directly under except above and before warning.